cocli/data/comid/templates/comid-cca-realm-refval.json

```diff
@@ -0,0 +1,60 @@
+{
+    "lang": "en-GB",
+    "tag-identity": {
+        "id": "43BBE37F-2E61-4B33-AED3-53CFF1428B16",
+        "version": 0
+    },
+    "entities": [
+        {
+            "name": "Workload Client Ltd.",
+            "regid": "https://workloadclient.example",
+            "roles": [
+                "tagCreator",
+                "creator",
+                "maintainer"
+            ]
+        }
+    ],
+    "triples": {
+        "reference-values": [
+            {
+                "environment": {
+                    "class": {
+                        "id": {
+                            "type": "uuid",
+                            "value": "CD1F0E55-26F9-460D-B9D8-F7FDE171787C"
+                        },
+                        "vendor": "Workload Client Ltd"
+                    },
+                    "instance": {
+                        "type": "bytes",
+                        "value": "QoS1aUymwNLPR4mguVrIAlyBjeUjBDZL580pgbLS7caFsyInfsJYGZYkE9jJssH1"
+                    }
+                },
+                "measurements": [
+                    {
+                        "value": {
+                            "integrity-registers": {
+                                "rim": {
+                                    "key-type": "text",
+                                    "value": [
+                                        "sha-384;QoS1aUymwNLPR4mguVrIAlyBjeUjBDZL580pgbLS7caFsyInfsJYGZYkE9jJssH1"
+                                    ]
+                                },
+                                "rem": {
+                                    "key-type": "text",
+                                    "value": [
+                                        "sha-384;IQe752H8pS2VE2oTVNt6TdV7Gya+DT2nHZ6yOYazS6YVq/ZRTPNeWp6lWgMtBop4",
+                                        "sha-384;JQe752H8pS2VE2oTVNt6TdV7Gya+DT2nHZ6yOYazS6YVq/ZRTPNeWp6lWgMtBop4",
+                                        "sha-384;MQe752H8pS2VE2oTVNt6TdV7Gya+DT2nHZ6yOYazS6YVq/ZRTPNeWp6lWgMtBop4",
+                                        "sha-384;NQe752H8pS2VE2oTVNt6TdV7Gya+DT2nHZ6yOYazS6YVq/ZRTPNeWp6lWgMtBop4"
+                                    ]
```

Comment:
I don't think this is the correct encoding.  The semantics associated with the multi-value digest is: _"same measured object using different hash algorithm"_.  Here we have multiple separate objects (i.e., REM[0], ..., REM[3]) that are measured using the same algorithm.  We should instead establish a convention to explicitly name the REMs.

Reply:
Maybe that's not entirely clear from the current prose, but that's the intended semantics:

_"Any digests associated with a measured object represent an acceptable state for the object."_

so, the encoding used above means: the measured object "rem" can have m[1] or m[2] or m[2] or m[3], which is not what we want to say.

There are four (or less, or more) distinct REM objects, each with its own set of acceptable values.

Reply:
If I understand correctly for `REM` can have 1-4 Measurements in any order is the flexibility provided by the system.

There is no REM-0 or REM-1, like that: It is just a collection with a 1-4 entries. 

I interpreted it based on the RMM spec hence I encoded it like this!

Reply:
no, REM banks have (implicit) names, i.e., their index.

It's not a collection, it's an ordered list.

When you do call the `RSI_MEASUREMENT_EXTEND` ABI you need to explicitly pass the index (that is, the name) of the bank which you want to extend.

Reply:
In particular, we need to be able to express semantics like: _"I only care that bank 2 is `0x3996003d486fb91ffb056f7d03f2b2992b215b31dbe7af4b373431fc7d319da3`"_ (if using SHA-256), or `11fb303c8d002309999c2ac9b6897bc294de3b76ab7664f024ec4b37552f0508b34f7390114bf09f93f73d508c57b0ef79d6b2fabbc0f3ad87f3c1ccef7d9be4` (if using SHA-512).

Reply:
I think, it is treated as a `Set` Not Individual entries, like REM-0. 
Also, as per our philosophy, Reference Values should be aligned as much close to how they appear in the Evidence.